src/app.ts

```
@@ -122,4 +122,11 @@ async function start() {
   sendBatchOK();
 }
 
-start();
+// Wrap in top level catch, to make sure that any errors are logged properly
+try {
+  start().catch((e) => {
+    log.error(e);
+  });
```

This catch swallows errors and allows continued execution. Did you mean to call process.exit(1)?

In an API, crashing the app makes it impossible to pass an error message to the consumer.

In this application we don't have a consumer in that sens and if we are stuck in an inconsitent state we might transfer corrupt data.